### PR TITLE
Multi select list teams tab members

### PR DESCRIPTION
**Pull request by Daniel-Wang**
cc @keybase/react-hackers @keybase/y2ksquad

### CODE REVIEW

File: shared/teams/team/rows/member-row/index.tsx [hunk 1]
```diff
@@ -66,63 +70,153 @@ export const TeamMemberRow = (props: Props) => {
   const roleLabel = !!active && !!props.roleType && typeToLabel[props.roleType]
   const isYou = props.you === props.username
 
-  const {showingPopup, setShowingPopup, toggleShowingPopup, popup, popupAnchor} = Kb.usePopup(attachTo => (
```
Comment (Daniel-Wang) [inline shared/teams/team/rows/member-row/index.tsx]: All this logic is refactored into line 170

File: shared/teams/team/rows/member-row/index.tsx [hunk 2]
```diff
@@ -191,21 +285,6 @@ export const TeamMemberRow = (props: Props) => {
             />
           )}
         </Kb.Box>
-        {flags.teamsRedesign && (
```
Comment (Daniel-Wang) [inline shared/teams/team/rows/member-row/index.tsx]: Refactored into line 170

File: shared/teams/team/rows/member-row/index.tsx [hunk 3]
```diff
@@ -66,63 +70,153 @@ export const TeamMemberRow = (props: Props) => {
...
+        onlyShowActionOnHover="fade"
+        height={Styles.isMobile ? 90 : 64}
+        icon={checkCircle}
+        iconStyleOverride={styles.checkCircle}
+        containerStyleOverride={styles.listItemMargin}
+        type="Large"
+        body={body}
+        firstItem={isOwner}
+        style={selected ? styles.selected : undefined}
+        onClick={isOwner ? undefined : () => onSelect(!selected)}
```
Comment (buoyad) [inline shared/teams/team/rows/member-row/index.tsx]: what is `isOwner` representing?

File: shared/teams/team/selection-popup.tsx [hunk 4]
```diff
@@ -15,25 +15,55 @@ const getSelectedCount = (state: Container.TypedState, props: Props) => {
...
   const dispatch = Container.useDispatch()
 
-  const onUnselect = () =>
-    dispatch(
-      TeamsGen.createSetChannelSelected({channel: '', clearAll: true, selected: false, teamID: props.teamID})
-    )
+  const onUnselect = () => {
+    switch (props.selectedTab) {
+      case 'channels':
+        return dispatch(
```
Comment (buoyad) [inline shared/teams/team/selection-popup.tsx]: Prefer
```
dispatch(...)
return
```

File: shared/teams/team/rows/member-row/index.tsx [hunk 5]
```diff
@@ -66,63 +70,153 @@ export const TeamMemberRow = (props: Props) => {
...
-      visible={showingPopup}
-      onHidden={() => setShowingPopup(false)}
-      closeOnSelect={true}
-      attachTo={attachTo}
-      position="bottom left"
-      positionFallbacks={['top left']}
-    />
-  ))
+  if (flags.teamsRedesign) {
+    const isOwner = true // TODO: Implement this
```
Comment (jakob223) [inline shared/teams/team/rows/member-row/index.tsx]: pass `role` in `props` and then make this `isOwner = props.role === 'owner'`

File: shared/teams/team/selection-popup.tsx [hunk 6]
```diff
@@ -62,15 +92,52 @@ const SelectionPopup = (props: Props) => {
...
+        </Kb.Box2>
+      )}
     </Kb.Box2>
   ) : null
 }
 
 export default SelectionPopup
 
 const styles = Styles.styleSheetCreate(() => ({
+  buttonStyle: {
```
Comment (buoyad) [inline shared/teams/team/selection-popup.tsx]: add `padding`/gap on desktop to the container instead

File: shared/teams/team/rows/member-row/index.tsx (hunk 3 above)
Comment (buoyad) [inline shared/teams/team/rows/member-row/index.tsx]: Also, how do we navigate to the member page? Maybe clicking the item body goes to the member page unless someone is already selected, then it toggles the selection?

File: shared/teams/team/rows/member-row/index.tsx [hunk 7]
```diff
@@ -66,63 +70,153 @@ export const TeamMemberRow = (props: Props) => {
...
+          ref={popupAnchor}
+          small={true}
+          tooltip="More actions"
+        />
+      </Kb.Box2>
+    )
+
+    return (
+      <Kb.ListItem2
+        action={actions}
```
Comment (buoyad) [inline shared/teams/team/rows/member-row/index.tsx]: There's a note in designs to hide these if any members are selected

File: shared/teams/team/rows/member-row/index.tsx (hunk 3 above)
Comment (Daniel-Wang) [reply shared/teams/team/rows/member-row/index.tsx]: Oh I'm changing this now so the owner is no different from everyone else, so gonna remove this stuff.

File: shared/teams/team/rows/member-row/index.tsx (hunk 7 above)
Comment (Daniel-Wang) [reply shared/teams/team/rows/member-row/index.tsx]: Cool I missed that! thanks

**Pull request merged**